spec/lib/water_drop/config_applier_spec.rb

```diff
@@ -33,6 +34,7 @@
         ack_timeout
         delivery_interval
         delivery_threshold
+        transactional_timeout
```

Comment:
This (and the `idempotent` and `transactional` booleans) will fail until delivery boy is updated, so this isn't able to be merged as-is